docs/portal/developer-portal/install/Install_the_UAN_Product_Stream.md

```diff
@@ -53,6 +53,10 @@ This section describes any UAN details that an administrator must be aware of be
 
 **Action**: Before executing this stage, the administrator must ensure that the UAN product tar file is in a media directory under `/etc/cray/upgrade/csm/`. When more than one product is being installed, place all the product tar files in the same directory.
 
+### deliver-product
+
+**Action**: Before executing this stage, the administrator must perform the prerequisites found in [Configuring a UAN for K3s](https://cray-hpe.github.io/docs-uan/en-273/advanced/enabling_k3s) when UAIs on UANs is enabled.
+
```

Comment:
Is there any reason why this is an external web link and not an internal cross reference link? The internal cross-reference link target would be  `../operations/Enabling_K3s.md`

Reply:
Updated the link to be internal.